### PR TITLE
Pylint

### PR DESCRIPTION
**Pull request by titilambert**

### CODE REVIEW

File: alignak/objects/trigger.py [hunk 1]
```diff
@@ -96,6 +96,7 @@ def compile(self):
         """
         self.code_bin = compile(self.code_src, "<irc>", "exec")
 
+    @staticmethod
```
Comment (Seb-Solon) [inline alignak/objects/trigger.py]: Trigget are still exec with trigger instance as far as I remember.

**Comment by Seb-Solon** [comment]
My second comment explinas why test fail IMH

**Comment by titilambert** [comment]
This one seems better :)

**Comment by Seb-Solon** [comment]
Related to #4, #87, #262 and naparuba/shinken#1393 .

**Pull request merged**